coordinates/sites.json

```diff
@@ -746,5 +746,1084 @@
+        "elevation_unit": "meter",
+        "longitude": 11.712500000000000,
+        "aliases": [
+            "Tautenburg 2 metre"
+        ]
+    },
+    "CATALINA61": {
+        "source": "SLALIB 2.5 obs.f plus additions",
+        "elevation": 2510.00,
+        "name": "Catalina 61 inch",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 32.416861111107799,
+        "elevation_unit": "meter",
+        "longitude": 249.268027778201628,
+        "aliases": [
+            "Catalina 61 inch"
+        ]
+    },
+    "STEWARD90": {
+        "source": "SLALIB 2.5 obs.f plus additions",
+        "elevation": 2071.00,
+        "name": "Steward 90 inch",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 31.963027778201631,
+        "elevation_unit": "meter",
+        "longitude": 248.400488888422672,
+        "aliases": [
+            "Steward 90 inch"
+        ]
+    },
+    "USSR6": {
+        "source": "SLALIB 2.5 obs.f plus additions",
+        "elevation": 2100.00,
+        "name": "USSR 6 metre",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 43.653333333333329,
+        "elevation_unit": "meter",
+        "longitude": 41.441666666666670,
+        "aliases": [
+            "USSR 6 metre"
+        ]
+    },
+    "ARECIBO": {
+        "source": "SLALIB 2.5 obs.f plus additions",
+        "elevation": 496.00,
+        "name": "Arecibo 1000 foot",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 18.343499999576146,
+        "elevation_unit": "meter",
+        "longitude": 293.246916666560708,
+        "aliases": [
+            "Arecibo 1000 foot"
+        ]
+    },
+    "CAMB5KM": {
+        "source": "SLALIB 2.5 obs.f plus additions",
+        "elevation": 17.00,
+        "name": "Cambridge 5km",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 52.170055555502579,
+        "elevation_unit": "meter",
+        "longitude": 0.043674999872843,
+        "aliases": [
+            "Cambridge 5km"
+        ]
+    },
+    "CAMB1MILE": {
+        "source": "SLALIB 2.5 obs.f plus additions",
+        "elevation": 17.00,
+        "name": "Cambridge 1 mile",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 52.163138888676961,
+        "elevation_unit": "meter",
+        "longitude": 0.039344444274902,
+        "aliases": [
+            "Cambridge 1 mile"
+        ]
+    },
+    "EFFELSBERG": {
+        "source": "SLALIB 2.5 obs.f plus additions",
+        "elevation": 366.00,
+        "name": "Effelsberg 100 metre",
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 50.524611111217077,
+        "elevation_unit": "meter",
+        "longitude": 6.883750000000000,
+        "aliases": [
+            "Effelsberg 100 metre"
+        ]
+    },
+    "GBT": {
```

Comment:
This entry is the same observatory with line 675 but has a more accurate coordinate. In my opinion, we should keep the more accurate one.